analytics.js

```diff
@@ -196,13 +196,35 @@ class Analytics {
         );
       });
 
+      let staging_env = false; // by default staging env is set to false
```

Comment:
We can re-define it to be string, "suffix"
"" - for production
"-staging" - staging

Reply:
done